recipe/meta.yaml

```diff
@@ -22,11 +22,16 @@ requirements:
     - setuptools
     - winpty
     - backports.shutil_which  # [py2k]
+    # use the m2w64 toolchain for Python 2.7 as VS 2008 is not supported
+    - m2w64-toolchain         # [py2k]
+    - libpython               # [py2k]
   run:
     - backports.shutil_which  # [py2k]
     - setuptools  # [py2k]
     - python
     - winpty
+    # the gcc libraries are only needed on win-32
+    - m2w64-gcc-libs          # [py2k and win32]
```

Comment:
You should not take this risk, it's only *by chance* of what how the code is compiled and what functions are called that this is the case. This library is far from a heavy dependency and should be added *every* time `m2w64-toolchain` is used as a compiler.

Reply:
Also, IMHO you should just compile it with `m2w64-toolchain` always. Using two different runtimes introduces the unnecessary risk of behavior changes.